components/schema-migrator/migrations/director/202012081537_ord_service_initial.up.sql

```diff
@@ -0,0 +1,302 @@
+                                  ELSE NEW.version_deprecated END;
+    RETURN NEW;
+END
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER set_release_status_api_def
+    BEFORE INSERT OR UPDATE
+    ON api_definitions
+    FOR EACH ROW
+EXECUTE PROCEDURE set_release_status();
+
+CREATE TRIGGER set_release_status_event_def
+    BEFORE INSERT OR UPDATE
+    ON event_api_definitions
+    FOR EACH ROW
+EXECUTE PROCEDURE set_release_status();
+
+
+ALTER TABLE api_definitions
+    ADD COLUMN ord_id                VARCHAR(256), /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN short_description     VARCHAR(255), /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN system_instance_aware BOOLEAN,
+    ADD COLUMN api_protocol          api_protocol, /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN tags                  JSONB,
+    ADD COLUMN countries             JSONB,
+    ADD COLUMN api_definitions       JSONB, /* Array of URLs pointing to API specs */
+    ADD COLUMN links                 JSONB,
+    ADD COLUMN api_resource_links    JSONB,
+    ADD COLUMN release_status        release_status, /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN sunset_date           VARCHAR(256),
+    ADD COLUMN successor             VARCHAR(256),
+    ADD COLUMN changelog_entries     JSONB,
+    ADD COLUMN labels                JSONB;
+
+/* Dummy update in order to apply the trigger logic above */
+UPDATE api_definitions
+SET name = name;
+
+ALTER TABLE event_api_definitions
+    ADD COLUMN ord_id                VARCHAR(256), /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN short_description     VARCHAR(255), /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN system_instance_aware BOOLEAN,
+    ADD COLUMN changelog_entries     JSONB,
+    ADD COLUMN links                 JSONB,
+    ADD COLUMN tags                  JSONB,
+    ADD COLUMN countries             JSONB,
+    ADD COLUMN release_status        release_status, /* ORD Required, nullable due to backwards compatibility */
+    ADD COLUMN sunset_date           VARCHAR(256),
+    ADD COLUMN successor             VARCHAR(256),
+    ADD COLUMN event_definitions     JSONB, /* Array of URLs pointing to Event specs */
+    ADD COLUMN labels                JSONB;
+
+/* Dummy update in order to apply the trigger logic above */
+UPDATE event_api_definitions
+SET name = name;
+
+
+/* Views exposing JSON columns structured */
+
+CREATE VIEW links AS
+SELECT *
+FROM (SELECT id                AS package_id,
+             NULL::uuid        AS api_definition_id,
+             NULL::uuid        AS event_definition_id,
+             links.title       AS title,
+             links.url         AS url,
+             links.description AS description
+      FROM packages,
+           jsonb_to_recordset(packages.links) AS links(title TEXT, description TEXT, url TEXT)) AS package_links
```

Comment:
@DimitarPetrov do you remember if the `...AS package_links` is needed for something specific here?
A similar thing can be noticed on line 139, 166, 189, 223, 261, 285.

Reply:
Discussed - issue was that without alias the migration failed with:
` error: migration failed: subquery in FROM must have an alias (column 6) in line 97: BEGIN;`